java/src/main/java/com/google/cloud/dataproc/templates/util/TemplateConstants.java

```diff
@@ -546,6 +546,15 @@ public interface TemplateConstants {
   String SPARK_BIGTABLE_INSTANCE_ID = "spark.bigtable.instance.id";
   String SPARK_BIGTABLE_CATALOG = "catalog";
 
+  /** Spark GCS Deltalake to Iceberg Properties */
+  String DELTALAKE_INPUT_LOCATION = "deltalake.input.location";
```

Comment:
our convention to name the properties is:
<templatename>.<input/output>.<propertyname>